ansible-scylla-node/tasks/upgrade/upgrade_ubuntu16.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+# Ensure apt-transport-https is installed
+- name: Ensure apt-transport-https is installed
+  ansible.builtin.apt:
+    name: apt-transport-https
+    state: present
+  become: true
+
+
+# Add Scylla apt key
+- name: Add Scylla apt key
+  ansible.builtin.apt_key:
+    keyserver: hkp://keyserver.ubuntu.com:80
```

Comment:
same stuff with keyservers and repo urls, should be taken from some global var

Reply:
The idea behind this is not to define this manually (since this is "encapsulated" per template), that's why there's no variable to override.